### PR TITLE
Fix export for group contributor over 500 users

### PR DESCRIPTION
**Pull request by puranban**
### Addresses

- https://github.com/mapswipe/mapswipe/issues/839

### Changes

- Fix user export of group over 500.

### CODE REVIEW

File: community-dashboard/app/components/ItemSelectInput/index.tsx [hunk 1]
```diff
@@ -135,17 +141,19 @@ function ItemSelectInput<Name extends string>(props: ItemSelectInputProps<Name>)
 
     const [opened, setOpened] = useState(false);
     const [searchText, setSearchText] = useState<string>('');
-
     const debouncedSearchText = useDebouncedValue(searchText);
 
     const variables = useMemo(() => ({
         search: debouncedSearchText,
+        offset: 0,
+        limit: 5,
```
Comment (samshara) [inline community-dashboard/app/components/ItemSelectInput/index.tsx:149]: Let's use the defined variable

File: community-dashboard/app/views/UserGroupDashboard/index.tsx [hunk 2]
```diff
@@ -100,6 +104,30 @@ const FILTERED_USER_GROUP_STATS = gql`
...
+                items {
+                    userId
+                    username
+                    isActive
+                    totalMappingProjects
+                    totalSwipeTime
+                    totalSwipes
+                }
+            }
+        }
```
Comment (samshara) [inline community-dashboard/app/views/UserGroupDashboard/index.tsx:127]: we need to add `id` for `userGroup`

Comment (puranban) [reply community-dashboard/app/views/UserGroupDashboard/index.tsx:127]: we don't use `id` here, `userId` is used as a unique identifier

Comment (frozenhelium) [reply community-dashboard/app/views/UserGroupDashboard/index.tsx:127]: In that case, we probably need to update apollo cache to ise `userID` as key @tnagorra?

Comment (puranban) [reply community-dashboard/app/views/UserGroupDashboard/index.tsx:127]: We planned to implement this later.
Reference issue: https://github.com/mapswipe/mapswipe/issues/845

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**